scss/_alert.scss

```diff
@@ -31,6 +31,7 @@
   margin: 0;
   font-size: inherit;
   font-weight: inherit;
+  line-height: 1.125;
```

Comment:
Not sure about that, but since we inherit font-size, maybe we should inherit line-height too ?